numpyro/infer/util.py

```diff
@@ -870,4 +870,11 @@ def helpful_support_errors(site):
                 "a reparameterizer, e.g. "
                 f"numpyro.handlers.reparam(config={{'{name}': ProjectedNormalReparam()}})."
             )
+        if "circular" in support_name:
+            raise ValueError(
```

Comment:
raise a warning instead. In VonMises tests, you can skip the warning

Reply:
I've changed this to a UserWarning and reworded slightly. I presumed I could revert my changes to `test_distributions.py` in commit a3ca41e because `isinstance(constraint.circular, constraint.interval) == True` so it just defaults to generating values in that interval for the tests. The tests pass when I do this.

I haven't found instances in the tests where we'd need to skip the new warning. But then I haven't fetched and merged from `pyro-ppl/numpyro/master` recently so there may be changes to the tests which I'll make sure pass.